src/cluster.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from 'react';
+import * as MapboxGL from 'mapbox-gl';
```

Comment:
Not exactly, with @ types, every types inside the @ type directory is already loaded. So you can already use `mapboxgl` as is.
But sometimes, namespace are not exposed globally and you have to use the import for type definition...
Also typescript does elide this kind of import but I'm not sure exactly _when_ (https://github.com/Microsoft/TypeScript/issues/2812)